src/bundle/Resources/public/scss/_extra-actions.scss

```
@@ -177,3 +177,12 @@
         }
     }
 }
+
+// TO DO: remove when redesigning !!
```

Should it stay? If so:
```suggestion
// TODO: remove when redesigning !!
```

---

> Should it stay? If so:

yes, because after whole redesign we have to change it